lib/cocina/models/validators/catalog_links_validator.rb

```diff
@@ -36,6 +37,16 @@ def validate_catalog(catalog)
                                  "(only one allowed) #{refresh_catalog_links}"
         end
 
+        def validate_sort_key
```

Comment:
JSON scheme supports `dependentRequired` that allows for this sort of validation (if you have this field, then you also must have this other field). Wondering if that can be used with OpenApi.

Reply:
With any new keywords, we would need to update the generators as well?

Reply:
This looks like a blocker however: `openapi_parser` does not yet support OpenAPI 3.1. https://github.com/ota42y/openapi_parser/issues/152

Reply:
Why can't we have nice things?